02_prompt_engineering/prompt_engineering.md

```diff
@@ -452,20 +452,20 @@ based on what the model does.  Try to iterate as quickly as possible.
 ### Model Runners
 
 We've created model runners to simplify the process of executing prompts.
-Our `MistralRunner` uses the Mistral instruct model by default and allows
+Our `Lamini` uses the Mistral instruct model by default and allows
```

Comment:
This section needs to be updated or removed since it pertains to the Mistral runner.

E.g. I don't think we default to Mistral, and we do not `automatically wraps the user and system prompts
in Mistral's prompt template`